dll/win32/shell32/shlexec.cpp

```diff
@@ -1431,60 +1428,128 @@ static HRESULT shellex_run_context_menu_default(IShellExtInit *obj,
     return r;
 }
 
-static HRESULT shellex_load_object_and_run(HKEY hkey, LPCGUID guid, LPSHELLEXECUTEINFOW sei)
+namespace
```

Comment:
Might be of interest (for later) to have that thing available on the whole shell32 🤔 
But why does it need to reside in a nameless namespace?